pkg/apis/core/v1alpha1/conversions.go

```diff
@@ -193,3 +193,122 @@ func Convert_v1alpha1_ShootStatus_To_core_ShootStatus(in *ShootStatus, out *core
 
 	return nil
 }
+
+func Convert_v1alpha1_ProjectSpec_To_core_ProjectSpec(in *ProjectSpec, out *core.ProjectSpec, s conversion.Scope) error {
+	if err := autoConvert_v1alpha1_ProjectSpec_To_core_ProjectSpec(in, out, s); err != nil {
+		return err
+	}
+
+	if owner := out.Owner; owner != nil {
+	outer:
+		for i, member := range out.Members {
+			if member.Name == owner.Name && member.APIGroup == owner.APIGroup && member.Kind == owner.Kind {
+				mustAddOwnerRole := true
```

Comment:
`mustAddOwnerRole` is not required since you early exit the loop in 210 in case the owner role was found.

Reply:
Indeed, nice catch, thank you. Removed.